content/en/docs/concepts/overview/working-with-objects/conditions.md

```diff
@@ -0,0 +1,102 @@
+- **Status**: The state of the condition, which can be `True`, `False`, or `Unknown`.
+- **Reason**: String explaining why the condition is in a current state
+- **Message**: Message explaining details of the message
+- **Last Transition Time**: This represents the timestamp where condition last changed status
+
+## Example: Pod Conditions
+
+Pods, which are an integral part of the Kubernetes resources use conditions to indicate their state. Below is an example:
+
+```yaml
+status:
+  conditions:
+  - type: PodScheduled
+    status: "True"
+    lastTransitionTime: "2023-05-10T16:29:24Z"
+  - type: Initialized
+    status: "True"
+    lastTransitionTime: "2023-05-10T16:29:24Z"
+  - type: ContainersReady
+    status: "True"
+    lastTransitionTime: "2023-05-10T16:29:31Z"
+  - type: Ready
+    status: "True"
+    lastTransitionTime: "2023-05-10T16:29:31Z"
+```
+
+In this example, each condition provides specific information about the pod's status, such as whether it has been scheduled, initialized, or is ready to handle traffic.
+
+In example above, each condition provides information as to the status of the pod. Whether the pod has been scheduled, initialized and ready to accept different traffic. Also whether the container in the pod are ready to be started.
+
+## Purpose and Usage of Conditions
+
+Conditions being an integral part of Kubernetes play very important role by:
+
+* **Status Information**: Conditions provide an in-depth understanding of the existing cluster state or resources. They help the user or a machine understand what’s going on.
+
+* **Communication**: Conditions act as a form of communication inside the cluster. It helps various components like the kubelet, the scheduler talk to each other or custom controllers to send and receive messages.
+
+* **Activity Engine**: They help custom controllers to perform actions. For given conditions, controllers can make HTTP requests, call to cloud API, etc. Once a pod is unschedulable, a controller might create more resources, Websites, etc.
+
+* **Events**:
+
+* They are used to record a single one-time event.
+
+* **Examples**: Kubernetes uses events to report changes during the control plane’s request processing. It uses events for purely informational purposes, trying to help understand what’s going on in the cluster.
+
+Examples where Conditions are used:
+
+* `FrequentSyncError`: Indicates the resource Seeding controller is struggling to create.
+
+* `ComponentsInstalled`: Indicates that a desired state of components has been installed.
+
+* `Repairing`: Indicates that the controller has started repairing things.
+
+* `Mounted`: Indicates that a controller from the Cloud integration is successfully mounted on the node.
+
+* **Conditions**:
+
+* They provide an engineering statement regarding the running process.
+
+* **Defining Conditions**: When we design conditions, multiple things have to be cared upon.
+
+* **Polarity**: any unhealthy state should be `True`.
```

Comment:
> Condition types should indicate state in the “abnormal-true” polarity. For example, if the condition indicates when a policy is invalid, the “is valid” case is probably the norm, so the condition should be called “Invalid”.

That's quoting the API conventions, which I think you're paraphrasing here. However:

> For some conditions, True represents normal operation, and for some conditions, False represents normal operation. ("Normal-true" conditions are sometimes said to have "positive polarity", and "normal-false" conditions are said to have "negative polarity".) Without further knowledge of the conditions, it is not possible to compute a generic summary of the conditions on a resource.

That's also quoting the same API conventions.

We should at least comment in passing that the example from earlier has:
 - `PodScheduled`
 - `Initialized`
- `ContainersReady`
- `Ready`

which are all abnormal-false.

Reply:
@sftim  Could you explain this further getting confused on this feedback